templates/_devices.html

```diff
@@ -1,35 +1,30 @@
 <script src="//cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.15/lodash.min.js" type="text/javascript"></script>
 <script type="text/javascript">
-  (function () {
+  function fetchArchivedDeviceTypesAndGenerateTable() {
     window.jQuery
       .ajax({
-        url: "https://api.balena-cloud.com/device-types/v1/"
+        url: "https://api.balena-cloud.com/v6/application?$filter=is_host%20eq%201&$filter=is_archived%20eq%20true&$select=app_name",
+      })
+      .done(function (archivedDeviceTypes) {
+        var archivedDeviceTypesArr = _.map(archivedDeviceTypes.d, function (app) { return app.app_name });
```

Comment:
I wouldn't rely on the name here, cause it could potentially change. Let me know if you need some help to expand from the application to the device

Reply:
thanks. done!